community/modules/scripts/ramble-setup/README.md

```diff
@@ -0,0 +1,104 @@
+## Description
+
+This module will create a set of startup-script runners that will setup Ramble,
+and install Ramble’s dependencies.
+
+For more information about ramble, see: https://github.com/GoogleCloudPlatform/ramble
```

Comment:
nit: I would just add a few more words that tell what ramble is on this page. Maybe something like:

`Ramble is a multi-platform experimentation framework capable of driving software installation, acquiring input files, configuring experiments, and extracting results. For more information about ramble, see:`

This is nit so you can just `Ack` and close if you want but I think it would help to tell people just a little more.